pkg/util/azureclient/authz/remotepdp/client_test.go

```diff
@@ -0,0 +1,71 @@
+package remotepdp
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"net/http"
+	"testing"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/policy"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/runtime"
+	"github.com/Azure/azure-sdk-for-go/sdk/azidentity"
+
+	testhttp "github.com/Azure/ARO-RP/test/util/http"
+)
+
+func TestClientCreate(t *testing.T) {
+	endpoint := "https://westus.authorization.azure.net/providers/Microsoft.Authorization/checkAccess?api-version=2021-06-01-preview"
+	scope := "https://authorization.azure.net/.default"
+	cred, err := azidentity.NewClientSecretCredential("888988bf-86f1-31ea-91cd-2d7cd011db48", "clientID", "clientSecret", nil)
+	if err != nil {
+		t.Error("Unable to create a new PDP client")
+	}
+	NewRemotePDPClient(endpoint, scope, cred)
+}
```

Comment:
there doesn't seem to be any check in here, we should test that the resulting object has the proper fields. here there is not assertion on it

Reply:
added one just now.